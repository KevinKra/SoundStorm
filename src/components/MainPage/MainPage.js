import React, { Component } from "react";
import "./MainPage.scss";
import { PrimaryTracks } from "../PrimaryTracks/PrimaryTracks";
import SupportContent from "../SupportContent/SupportContent";
import SupportPage from "../SupportPage/SupportPage";
import Header from "../../containers/Header/Header";
import HeroImage from "../HeroImage/HeroImage";

export default class MainPage extends Component {
  render() {
    return (
      <section className="MainPage">
        <Header />
        <HeroImage
          height="325px"
          width="90vw"
          image={
            "https://images.unsplash.com/photo-1501612780327-45045538702b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
          }
        />
        <PrimaryTracks />
        <SupportContent />
        <SupportPage />
        <div className="audioBar-buffer" />
      </section>
    );
  }
}
