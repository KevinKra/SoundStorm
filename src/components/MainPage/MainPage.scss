@import "../../_styles/variables.scss";

.MainPage {
  display: flex;
  flex-direction: column;
  font-family: "Roboto", "sans-serif";
  background-color: $secondary-dark-color;
  align-items: center;
  position: relative;
  min-height: 100vh;
  width: 90vw;
  margin-left: 10vw;
  .audioBar-buffer {
    margin: 2rem;
  }
}
