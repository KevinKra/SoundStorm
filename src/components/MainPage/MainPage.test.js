import React from "react";
import MainPage from "./MainPage";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("<MainPage />", () => {
  let wrapper = shallow(<MainPage />);
  console.log(wrapper.debug());
  it("should match the snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
