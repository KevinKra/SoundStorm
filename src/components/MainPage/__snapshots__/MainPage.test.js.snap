// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainPage /> should match the snapshot 1`] = `
<section
  className="MainPage"
>
  <Connect(Header) />
  <HeroImage
    height="325px"
    image="https://images.unsplash.com/photo-1501612780327-45045538702b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
    width="90vw"
  />
  <PrimaryTracks />
  <SupportContent />
  <Connect(SupportPage) />
  <div
    className="audioBar-buffer"
  />
</section>
`;
